<launch>

  <node name="main_state" pkg="main_state" type="main.py" output="screen" args="RoboZoo" />

  <node name="circle_detection" pkg="faces" type="circle.py">
    <remap from="image" to="/external_cam/image_raw"/>
    <remap from="output" to="/circle_detection/output"/>
  </node>

  <node name="circle_output" pkg="image_view" type="image_view">
    <remap from="image" to="/circle_detection/output"/>
  </node>

  <node name="static_pub" pkg="rostopic" type="rostopic" args="pub /voice/output std_msgs/String hello "/>
</launch>
